client/a8c-for-agencies/sections/marketplace/pressable-overview/plan-selection/details.tsx

```diff
@@ -1,3 +1,4 @@
+import { isEnabled } from '@automattic/calypso-config';
 import { Button } from '@automattic/components';
```

Comment:
Changed back to `'@automattic/components'` as more changes are needed and it can be done in another PR.